### PR TITLE
BIP 132: Add Committee-based BIP Acceptance Process

### PR DESCRIPTION
**Pull request by andychase**
Previous discussions:

* https://gist.github.com/andychase/dddb83c294295879308b
* https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-September/010948.html
* https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-September/010914.html

I would like to be assigned a number please! Thanks.

### CODE REVIEW

File: bip-andychase.mediawiki [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+A BIP Process Manager should be chosen who is in charge of:
+
+* Declaring where and how BIPs, Opinions, and Committee Declaration should be posted and updated officially.
+* Maintaining the security and authenticity of BIPs, Opinions, and Committee Declarations
+* Publishing advisory documents about what kinds of proof of stakes are valid and what kinds should be rejected.
+* Naming a series of successors for the roles of the BIP Process Manager and BIP Editor (BIP-001) as needed.
+
+== Conditions for activation ==
+
+In order for this process BIP to become active, it must succeed by its own rules. At least a 4% sample of the Bitcoin community must be represented, with at least one committee in each segment included. Once at least one committee has submitted a declaration, a request for comments will be called and the process should be completed from there.
```
Comment (luke-jr) [inline bip-andychase.mediawiki]: Until this BIP is active, its rules do not apply, so this would be a form of circular reasoning. I like the idea of putting conditions for activation in the BIP text, but I don't think we can just let the author set any conditions they like either...

Comment (andychase) [inline bip-andychase.mediawiki]: The idea here is that if the system doesn't even work for itself, it shouldn't activate. Currently the conditions are a bit fuzzy with some flaws, so I'd like to narrow those down before changing the BIP from being a draft and making it eligible for succeeding by its own rules.

Comment (luke-jr) [inline bip-andychase.mediawiki]: Ok, so these are *additional* conditions? Can you clarify this in the draft?

**Comment by btcdrak** [comment]
@andychase The whole concept of "acceptance by committee" is wrong and invalidates much of this BIP see https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-September/010955.html

tl;dr: The status of a BIP is defined by it's adoption, not by a committee.

**Comment by andychase** [comment]
@btcdrak Thanks for your continued feedback. Can you send your comment to the mailing list? I'd love for it to see a wider audience.

**Comment by andychase** [comment]
After some thought I'd like to raise an objection to the BIP-2 number assignment. I think BIPS 1-9 should be specifically reserved for a BIPS that proposes to replace BIP-1 in its entirety.

This one is for a specific part of the BIP lifecycle; and will hopefully inspire other ones similar.
